config/migrations/20200923144227-notule-cleanup0.sparql

```diff
@@ -0,0 +1,20 @@
+PREFIX dossier: <https://data.vlaanderen.be/ns/dossier#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX brc: <http://kanselarij.vo.data.gift/id/concept/beslissings-resultaat-codes/>
+PREFIX type: <http://kanselarij.vo.data.gift/id/concept/document-type-codes/>
+
+INSERT {
+  GRAPH ?g {
+      ?collection ext:documentType type:e149294e-a8b8-4c11-83ac-6d4c417b079b.
+  }
+}
+WHERE {
+    GRAPH ?g {
+      ?not a <http://mu.semte.ch/vocabularies/ext/Notule>.
+      ?not ext:getekendeDocumentVersiesVoorNotulen ?doc.
+      ?doc a ?doctype.
```

Comment:
Hier doe je eigenlijk niet meer dan testen of `?doc` een type heeft ... op zich niet verkeerd, maar je bedoelde ws iets anders? (hieronder nog eentje ...)

Reply:
klopt, ik had dit bij het aanmaken lokaal geprint om te zien dat het de juiste was, maar is idd niet meer nodig. Ik haal ze er uit.